lib/test_process.dart

```diff
@@ -28,15 +28,13 @@ class TestProcess {
 
   /// A [StreamQueue] that emits each line of stdout from the process.
   ///
-  /// A copy of the underlying stream can be retreived using [stdoutStream].
-  StreamQueue<String> get stdout => _stdout;
-  StreamQueue<String> _stdout;
+  /// A copy of the underlying stream can be retrieved using [stdoutStream].
+  late final StreamQueue<String> stdout = StreamQueue(stdoutStream());
```

Comment:
@natebosch – should we care about the whole `late final` creates a setter thingy here? Keep the properties?

Reply:
`late final` here is great! nevermind.

https://github.com/dart-lang/language/blob/master/accepted/future-releases/nnbd/feature-specification.md#late-fields-and-variables